settings.yaml

```diff
@@ -12,16 +12,26 @@ end_year: 2018
 land_use_freq: 1
 travel_model_freq: 10
 
-# simulation platforms (comment out to turn off)
+# simulation platforms (leave blank to turn off)
 land_use_model: urbansim
-travel_model: beam
-activity_demand_model: activitysim
+travel_model: polaris
+activity_demand_model: polaris
+
+# docker or singularity
+container_manager: docker
+#singularity settings
+singularity_images:
+  urbansim: block_model_v2_pb.sif
+  beam:
+  activitysim:
+  polaris:
 
 # docker settings
 docker_images:
   urbansim: mxndrwgrdnr/block_model_v2_pb:latest
-  beam: dimaopen/beam:0.8.6.3
-  activitysim: mxndrwgrdnr/activitysim:latest
+  beam: 
```

Comment:
I think this was probably a mistake...can you put the image names back in here?

Reply:
Done